GetSubscriptionKeys/handler.ts

```diff
@@ -0,0 +1,124 @@
+import { Context } from "@azure/functions";
+
+import * as express from "express";
+
+import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
+import {
+  AzureApiAuthMiddleware,
+  IAzureApiAuthorization,
+  UserGroup
+} from "io-functions-commons/dist/src/utils/middlewares/azure_api_auth";
+import { ContextMiddleware } from "io-functions-commons/dist/src/utils/middlewares/context_middleware";
+import {
+  withRequestMiddlewares,
+  wrapRequestHandler
+} from "io-functions-commons/dist/src/utils/request_middleware";
+
+import { ApiManagementClient } from "@azure/arm-apimanagement";
+import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
+import { toError } from "fp-ts/lib/Either";
+import { TaskEither, tryCatch } from "fp-ts/lib/TaskEither";
+import {
+  IResponseErrorInternal,
+  IResponseErrorNotFound,
+  IResponseSuccessJson,
+  ResponseErrorInternal,
+  ResponseErrorNotFound,
+  ResponseSuccessJson
+} from "italia-ts-commons/lib/responses";
+import { ServiceId } from "../generated/definitions/ServiceId";
+import { ServiceIdMiddleware } from "../utils/middlewares/serviceid";
+
+const azureSubscriptionId = getRequiredStringEnv("AZURE_SUBSCRIPTION_ID");
+const servicePrincipalClientId = getRequiredStringEnv(
+  "SERVICE_PRINCIPAL_CLIENT_ID"
+);
+const servicePrincipalSecret = getRequiredStringEnv("SERVICE_PRINCIPAL_SECRET");
+const servicePrincipalTenantId = getRequiredStringEnv(
+  "SERVICE_PRINCIPAL_TENANT_ID"
+);
+const azureApimResourceGroup = getRequiredStringEnv(
+  "AZURE_APIM_RESOURCE_GROUP"
+);
+const azureApim = getRequiredStringEnv("AZURE_APIM");
+
+function getApiClient(): TaskEither<Error, ApiManagementClient> {
+  return tryCatch(
+    () =>
+      msRestNodeAuth.loginWithServicePrincipalSecret(
+        servicePrincipalClientId,
```

Comment:
these should be passed as parameters (only the `main()` has access to global variables)